### DISCUSSION TITLE
Disable vApp options

### DISCUSSION THREAD

**Discussion by NathanDotTo** [vmware/govmomi#2886] [Community Help]
Hi

Is there a means to achieve the same logical outcome of disabling vApp options as illustrated in the attached image please?

Thanks

Nathan

![Screenshot 2022-06-21 at 18 55 09](https://user-images.githubusercontent.com/13392790/174855891-583917da-5fd7-4532-8a52-4a4e71dfde89.png)

**Comment by james-martinez** [comment]
vAppConfigRemoved: true ?

https://vdc-repo.vmware.com/vmwb-repository/dcr-public/1ef6c336-7bef-477d-b9bb-caa1767d7e30/82521f49-9d9a-42b7-b19b-9e6cd9b30db1/vim.vm.ConfigSpec.html
